### ISSUE TITLE
Checkpoint 10: 1st song is not behaving like the others

### ISSUE THREAD

**Issue by rapid99** [rapid99/bloc-jams-angular#23]
Situation: In checkpoint 10, I implemented functionality that allowed the thumb to track how far along the song was. This works as expected for all songs except for the first song. It also only happens AFTER you initially click on the thumb

Error message: Failed to set the 'currentTime' property on 'HTMLMediaElement': The provided double value is non-finite. 

My question: why does this only occur on the first song? There was never any code inserted that would have singled out this one song. 

Best guess: It might have something to do with when the elements and rules for starting this behavior are implemented (and it isn't being applied to when the page loads but rather after a song has begun playing

**Comment by bdougie** [comment]
I definitely see the errors and noticed that you switch between using the [album and playerBar](https://github.com/rapid99/bloc-jams-angular/blob/master/app/templates/player_bar.html#L19-L23) objects to get and set data. Do you think you could get data from just the playeBar?

Also let me know if this S/O question provides any insight on the error as well http://stackoverflow.com/questions/34797312/html5-audio-element-seek-slider-failed-to-set-the-currenttime-property-on

**Comment by rapid99** [comment]
Good catch. I will redo this to make it consistent across the board and see how it works. We're meeting today this afternoon so I will let you know how it goes then.

**Comment by bdougie** [comment]
Cool, @ more things. Your indentation and spacing is off: `thumbStyle()` should not be nested `trackThumb()` https://github.com/rapid99/bloc-jams-angular/blob/master/app/scripts/directives/seekBar.js#L65-L67

Also you need another `notifyOnChange(scope.value);` so you don't have to click the thumb directly to track it.
